models/stg_quickbooks__bill_payment_line.sql

```diff
@@ -31,10 +31,10 @@ fields as (
 final as (
     
     select 
-        cast(bill_payment_id as {{ dbt_utils.type_int() }}) as bill_payment_id,
+        cast(bill_payment_id as {{ dbt_utils.type_string() }}) as bill_payment_id,
         index,
         amount,
-        cast(bill_id as {{ dbt_utils.type_int() }}) as bill_id,
+        cast(bill_id as {{ dbt_utils.type_string() }}) as bill_id,
         deposit_id,
         expense_id,
         journal_entry_id,
```

Comment:
It looks like you also changed the macro to specify `deposit_id`, `expense_id`, `journal_entry_id`, `linked_bill_payment_id` and `vendor_credit_id` as datatype string, however, you don't explicitly do the cast as you've done for `bill_id` and `bill_payment_id`, why is that?